lib/plugins/sfDoctrinePlugin/lib/task/sfDoctrineBuildModelTask.class.php

```diff
@@ -77,7 +77,7 @@ protected function execute($arguments = array(), $options = array())
       $code = file_get_contents($file);
 
       // introspect the model without loading the class
-      if (preg_match_all('/@property (\w+) \$(\w+)/', $code, $matches, PREG_SET_ORDER))
+      if (preg_match_all('/@property (\S+) \$(\w+)/', $code, $matches, PREG_SET_ORDER))
```

Comment:
What does this change exactly do? Consider to comment about this change (with examples) in the commit message body.